tendermint/Cargo.toml

```diff
@@ -16,7 +16,7 @@ log = "0.4"
 
 nimiq-block = { path = "../primitives/block" }
 nimiq-hash = { path = "../hash" }
-nimiq-primitives = { path = "../primitives", features = ["policy"] }
+nimiq-primitives = { path = "../primitives", features = ["policy", "validators"] }
```

Comment:
It seems to me that this could be better fixed at a lower layer:

AFAICT, the `tendermint` crate currently can't compile standalone because it [depends on the `block` crate](https://github.com/nimiq/core-rs-albatross/blob/albatross/tendermint/Cargo.toml#L17), and the `block` crate is the one that actually requires the `validators` feature to be enabled on the `primitives` crate non-conditionally (f.e. see [here](https://github.com/nimiq/core-rs-albatross/blob/albatross/primitives/block/src/macro_block.rs#L67) ), so the problem could be fixed (for both, the `block` and `tendermint` crates) by enabling the `validators` when the `block` crate depends on the `primitives` crate [here](https://github.com/nimiq/core-rs-albatross/blob/albatross/primitives/block/Cargo.toml#L34) instead of doing it here at a higher layer.

Wdyt?

Reply:
You're right. I already changed it.